source/_integrations/tesla_fleet.markdown

```diff
@@ -143,6 +147,8 @@ These are the entities available in the Tesla Fleet integration. Not all entitie
 |Binary sensor|Backup capable|Yes|
 |Binary sensor|Grid services active|Yes|
 |Binary sensor|Grid services enabled|Yes|
+|Number|Backup reserve|Yes|
```

Comment:
what is meant by reserve?

Reply:
https://www.tesla.com/en_au/support/energy/powerwall/mobile-app/backup-reserve

Reply:
Thank you